lib/models/fee_options/fee_option.dart

```diff
@@ -32,9 +32,7 @@ abstract class FeeOption {
     }
   }
 
-  bool isAffordable({required int balance, required int amountSat}) {
-    return (amountSat + txFeeSat) < balance;
-  }
+  bool isAffordable({int? balance, int walletBallance, required int amountSat});
```

Comment:
NIT: 
balance => lightningBalance 
walletBalance =< onchainBalance

Reply:
These are the naming we use on app:
https://github.com/breez/c-breez/blob/0895c06664119fc9bdb8095a1f6da6611648fa15/lib/bloc/account/account_state_assembler.dart#L26-L27